.mergify.yml

```diff
@@ -0,0 +1,72 @@
+pull_request_rules:
+  - name: remove outdated reviews
+    conditions:
+      - base=master
+    actions:
+      dismiss_reviews:
```

Comment:
I see we also have this on firrtl, but what does this rule do? A naive reading would be that it always just dismisses reviews on master but it clearly doesn't do that...

Reply:
Looks like it enables mergify to take the last approved review; a stale review isn't considered if there is another approving one. https://doc.mergify.io/examples.html?highlight=dismiss#id11